src/card.css

```diff
@@ -0,0 +1,105 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card--main {
+  border: 1px solid var(--border-color);
+  border-radius: 5px;
+  width: 200px;
+  background-color: var(--background-color);
+}
+
+.card__pic_container {
+  display: flex;
+  padding-top: 32px;
+  width: 160px;
+}
+
+.card__pic {
+  width: 160px;
+  height: 134px;
+}
+
+.card__about {
+  display: flex;
+  width: var(--section-width);
+  margin-top: 40px;
+}
+
+.card__about--font {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--font-color);
+}
+
+.card__prod_id {
+  width: 99px;
+  padding-top: 4px;
+  align-self: flex-start;
+  margin-left: 16px;
+}
+
+.card__prod_id--font {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--card-font);
+}
+
+.card__cost {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-top: 24px;
+  width: var(--section-width);
+}
+
+.card__cost--font {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: var(--card-font);
+}
+
+.card__cost_value {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: var(--font-color);
+}
+
+.card__button {
+  box-sizing: border-box;
```

Comment:
It is a good practice to set this property for all elements(*) and don't change without extra need

Reply:
realized) thank!